src/angularComponent/ngcOmniboxController.js

```diff
@@ -145,6 +145,25 @@ export default class NgcOmniboxController {
     this.highlightedIndex = -1; // -1 means nothing is highlighted
   }
 
+  /**
+   * Highlights a particular suggestion item.
+   *
+   * @param {Object} item
+   */
+  highlightItem(item) {
+    const match = this._suggestionsUiList.find((listItem) => listItem.data === item);
+
+    if (match) {
+      this.highlightedIndex = match.index;
```

Comment:
Is it worth saving `match` to `this`? Then we don't have to loop again for `isHighlighted()`.